akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -340,7 +340,6 @@ object MediaTypes extends ObjectRegistry[(String, String), MediaType] {
   val `application/grpc+proto`                                                    = abin("grpc+proto", NotCompressible)
   val `application/lha`                                                           = abin("lha", NotCompressible, "lha")
   val `application/lzx`                                                           = abin("lzx", NotCompressible, "lzx")
-  val `application/mspowerpoint`                                                  = abin("mspowerpoint", NotCompressible, "pot", "pps", "ppt", "ppz")
```

Comment:
I'm afraid it looks like this one cannot be removed without introducing a binary incompatiblity (that is what the build failure says) - could you reintroduce it, but mark it deprecated?